CHANGELOG.md

```diff
@@ -2,16 +2,143 @@
+| `vcenterreceiver` | `otelcol/vcenter` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver` |  ([#34449](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34449)) |
+| `redisreceiver` | `otelcol/redisreceiver` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver` |  ([#34470](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34470)) |
+| `routingprocessor` | `otelcol/routing` | `github.com/open-telemetry/opentelemetry-collector-contrib/processor/routingprocessor` |  ([#34550](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34550)) |
+| `solacereceiver` | `otelcol/solacereceiver` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/solacereceiver` |  ([#34466](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34466)) |
+| `splunkenterprisereceiver` | `otelcol/splunkenterprisereceiver` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver` |  ([#34452](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34452)) |
+| `statsdreceiver` | `otelcol/statsdreceiver` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/statsdreceiver` |  ([#34547](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34547)) |
+| `tailsamplingprocessor` | `otelcol/tailsampling` | `github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor` |  ([#34550](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34550)) |
+| `sqlserverreceiver` | `otelcol/sqlserverreceiver` | `github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sqlserverreceiver` |  ([#34451](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34451)) |
+
+- (Contrib) `elasticsearchreceiver`: Enable more index metrics by default ([#34396](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34396))
+  This enables the following metrics by default:
+  `elasticsearch.index.documents`
+  `elasticsearch.index.operations.merge.current`
+  `elasticsearch.index.segments.count`
+  To preserve previous behavior, update your Elasticsearch receiver configuration to disable these metrics.
+- (Contrib) `vcenterreceiver`: Enables all of the vSAN metrics by default. ([#34409](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34409))
+  The following metrics will be enabled by default now:
+  - vcenter.cluster.vsan.throughput
+  - vcenter.cluster.vsan.operations
+  - vcenter.cluster.vsan.latency.avg
+  - vcenter.cluster.vsan.congestions
+  - vcenter.host.vsan.throughput
+  - vcenter.host.vsan.operations
+  - vcenter.host.vsan.latency.avg
+  - vcenter.host.vsan.congestions
+  - vcenter.host.vsan.cache.hit_rate
+  - vcenter.vm.vsan.throughput
+  - vcenter.vm.vsan.operations
+  - vcenter.vm.vsan.latency.avg
```

Comment:
```suggestion
  - vcenter.vm.vsan.latency.avg
- (Contrib) `vcenterreceiver`: Several host performance metrics now return 1 data point per time series instead of 5. ([#34708](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34708))
  The 5 data points previously sent represented consecutive 20s sampling periods. Depending on the collection interval 
  these could easily overlap. Sending just the latest of these data points is more in line with other performance metrics.
  
  This change also fixes an issue with the googlecloud exporter seeing these datapoints as duplicates.
  
  Following is the list of affected metrics which will now only report a single datapoint per set of unique attribute values.
  - vcenter.host.cpu.reserved
  - vcenter.host.disk.latency.avg
  - vcenter.host.disk.latency.max
  - vcenter.host.disk.throughput
  - vcenter.host.network.packet.drop.rate
  - vcenter.host.network.packet.error.rate
  - vcenter.host.network.packet.rate
  - vcenter.host.network.throughput
  - vcenter.host.network.usage
```